### DISCUSSION TITLE
Branch name autogenerated

### DISCUSSION THREAD

**Discussion by FacuV** [community/community#53221] [Codespaces]
### Select Topic Area

General

### Body

Dear GitHub Support Team,

I hope you are doing well. I am reaching out to you as a user of GitHub Codespaces to request a new feature that I believe would be of great value to many users.

My suggestion is to add an option in Codespaces that allows for the automatic generation of branch names when creating an issue. Currently, users must manually enter branch names, which can be a tedious and error-prone process. Automating this process would reduce errors and save valuable time.

I believe this feature would be very useful for developers who use Codespaces in their daily work. If implemented, it would allow users to focus on actual work instead of worrying about branch management.

Thank you for your attention to this request, and I look forward to hearing back from you. If you need further information or have any questions, please do not hesitate to contact me.

Sincerely, Facundo

**Comment by dmilburn** [comment]
Hi @FacuV,

Thanks for reaching out. A couple of thoughts here:

- I think this might be a better for for the Issues team since it's about a side effect taking place upon creation of an issue. There's a separate category here: https://github.com/orgs/community/discussions/categories/issues-and-projects
- If you did want to reach out to the GitHub support team for assistance, please go [here](https://support.github.com/). This is intended as a less official public forum where community members can help out as well. 
- One could build this themselves using github actions. See a similar action here: https://github.com/peterjgrainger/action-create-branch
